web/client/components/map/cesium/plugins/WMSLayer.js

```diff
@@ -102,6 +102,9 @@ function wmsToCesiumOptions(options) {
             headers,
             proxy: proxy && new WMSProxy(proxy) || new NoProxy()
         }),
+        // #7516 this helps Cesium to use CORS requests in a proper way, even when headers are not
+        // present in the Resource
+        tileDiscardPolicy: options.noCors ? undefined : new Cesium.NeverTileDiscardPolicy(),
```

Comment:
let's use options.tileDiscardPolicy :

- tileDiscardPolicy = undefined -> NeverTileDiscardPolicy -- comment the reason of using this as default  in this line, as you did, linking the issue and the permalink to the code, so it is clear why we force it. 
- tileDiscardPolicy = 'none' -> undefined -- use defaults of cesium
Open for other customizations, as object with configuration: 
https://cesium.com/downloads/cesiumjs/releases/1.17/Build/Documentation/DiscardMissingTileImagePolicy.html

Permalink to code of cesium: https://github.com/CesiumGS/cesium/blob/48c8abaca6257b568edc8c992ede5f2c487cdd38/Source/Scene/ImageryProvider.js#L366